### PR TITLE
Params for filtering resources on language or category.

### PR DESCRIPTION
**Pull request by heavysaturn**
This adds an endpoint that returns all resources for a specified language.

This resolves one of the issues in #14

### CODE REVIEW

**Comment by aaron-suarez** [comment]
I would like to hold off on merging this until we have a conversation here https://github.com/OperationCode/resources_api/issues/14#issuecomment-430845378.

@heavysaturn Please feel free to contribute to that conversation!

**Comment by heavysaturn** [comment]
I agree, params for resources is a better solution, I'll wait for #21 to merge and then implement that.

**Comment by heavysaturn** [comment]
Updated to solve #29. Implementation is fully case insensitive, and also supports both filters in a single query.

File: app/api/routes.py [hunk 1]
```diff
@@ -48,9 +49,58 @@ def get_resource(id):
...
 def get_resources():
+    """
+    Gets a paginated list of resources.
+
+    If the URL parameters `language` or `category` are found
+    in the request, the list will be filtered by these parameters.
+
+    The filters are case insensitive.
+    """
+    resource_list = []
```
[the remark this answers is not in the document]
Comment (heavysaturn) [reply app/api/routes.py]: yep, you're right. removing.

**Comment by heavysaturn** [comment]
GitHub refuses to update this PR with the commit I made to address your review, even though the commit clearly exists on my branch. I'll try to open a new PR.

**Pull request closed without merging**